.github/workflows/chromatic.yml

```diff
@@ -14,20 +14,30 @@ jobs:
         # Operating System
         runs-on: ubuntu-latest
 
+        strategy:
+          matrix:
+            node-version: [ 20.x ]
+
         # Job steps
         steps:
             - uses: actions/checkout@v3
               with:
                   fetch-depth: 0 # 👈 Required to retrieve git history
 
+            - name: Setup Environment
+              uses: prezly/setup-github-actions@v1
+              with:
+                node: ${{ matrix.node-version }}
+                pnpm: "9.15.4"
+
             - name: Install dependencies
-              run: npm ci
+              run: pnpm install
```

Comment:
Should we use `--frozen-lockfile` here?

Reply:
https://github.com/prezly/theme-kit-js/pull/844/commits/4b9e4875d7cdf564173fbb51e3ad967c64a34de0